python/src/opendp/_data.py

```diff
@@ -25,6 +29,31 @@
 ]
 
 
+def _onceframe_extract_lazyframe(
```

Comment:
This is used in turn by `_lazyframe`, but that function doesn't seem to have any usages. Do we either/both of these?